docs/apps/features/04-airdrops.mdx

```diff
@@ -114,3 +114,8 @@ and campaign-specific items like an eligibility criteria link.
 |                                          |
 | ---------------------------------------- |
 | ![All](/screenshots/drops-settings.webp) |
+
+#### Geo-fencing / Geographical Restrictions
```

Comment:
Do you not want to use an upper level heading to highlight it in the sidebar menu?

Reply:
Hmm yeah that's a good idea. At first I was thinking it's related to advanced-settings, but better to highlight it in the table of contents so people can find it faster. Would you mind editing that before merging?